pages/entity/_type/_.vue

```diff
@@ -11,7 +11,7 @@
     v-else
     data-qa="entity page"
   >
-    <b-row class="flex-md-row">
+    <b-row class="flex-md-row pt-3">
```

Comment:
This is not sufficient, having a padding-top on entity or search pages. Eg:
/great-women-in-eu-history

There should be a margin between the header menu and the underlying content on every page where there is no hero banner.

Reply:
I was going to target the main navigation, but as you mentioned we would have to stop any margins on the homepage. Sadly there is no `homepage` class or ID I can target to achieve this. Can you please let me know your thoughts?

Reply:
And a hero banner can be added to any browse page. What I would like to suggest for now is give the main navigation a mb-3 as a default. And add these lines to the hero.scss:

````.navbar + div .jumbotron {
      margin-top: -1rem;
}
`````
I think that should to the trick.